scripts/compose.py

```diff
@@ -1081,15 +1096,29 @@ def build_candidate_manifest(self):
         return self.bc["projects"]["logstash-docker"]["packages"][key]
 
     def _content(self):
+        self.es_urls = self.options.get("logstash_elasticsearch_urls")
+        if not self.es_urls:
+            self.es_urls = ["http://elasticsearch:9200"]
         return dict(
             depends_on={"elasticsearch": {"condition": "service_healthy"}} if self.options.get(
                 "enable_elasticsearch", True) else {},
-            environment={"ELASTICSEARCH_URL": "http://elasticsearch:9200"},
+            environment={
+                "ELASTICSEARCH_URL={}".format(self.es_urls),
```

Comment:
can this handle multiple urls?  if so, probably need a `",".join(self.es_urls)`, otherwise, maybe just take the first entry?

Reply:
self.es_urls is a String, so in case it has multiple URL it'd be something like this 'url1,url2,url3,url4' so the `.format(self.es_urls)` would put the whole value in the formated string.

Reply:
It looks like a list, a test would help demonstrate.

Reply:
I get you now, those options are arrays, not strings